src/components/Navigation.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { NavLink } from 'react-router-dom'
+import { ReactComponent as Home } from '../icons/home.svg'
```

Comment:
Is it the required way of importing components for SVG's? Wouldn`t "import Home from '../icons/home.svg' work as well?

Reply:
I think that would work as well. This way had the benefit that it was instantly a component that I could style.